src/libs/actions/IOU.ts

```diff
@@ -2857,7 +2857,7 @@ type UpdateMoneyRequestTaxRateParams = {
 function updateMoneyRequestTaxRate({transactionID, optimisticReportActionID, taxCode, taxAmount, policy, policyTagList, policyCategories}: UpdateMoneyRequestTaxRateParams) {
     const transactionChanges = {
         taxCode,
-        ...(taxAmount && {taxAmount}),
+        ...(taxAmount !== null && {taxAmount}),
```

Comment:
```suggestion
        taxAmount,
```

Reply:
@MonilBhavsar The type of `taxAmount` is number and we also set a default value for this field is 0 so I think we don't need null check here